services/horizon/internal/flags.go

```
@@ -194,12 +195,23 @@ func Flags() (*Config, support.ConfigOptions) {
 			ConfigKey:   &config.EnableCaptiveCoreIngestion,
 		},
 		&support.ConfigOption{
-			Name:        "exp-enable-ingestion-filtering",
+			Name:        EnableIngestionFilteringFlag,
 			OptType:     types.Bool,
-			FlagDefault: false,
+			FlagDefault: true,
 			Required:    false,
-			Usage:       "causes Horizon to enable the experimental Ingestion Filtering and the ingestion admin HTTP endpoint at /ingestion/filter",
 			ConfigKey:   &config.EnableIngestionFiltering,
+			CustomSetValue: func(opt *support.ConfigOption) error {
+				if val := viper.GetString(opt.Name); val != "" {
+					stdLog.Printf(
+						"DEPRECATED - %s causes Horizon to enable the experimental Ingestion Filtering and the "+
+							"ingestion admin HTTP endpoint at /ingestion/filter. Ingestion filtering is now enabled by "+
+							"default and this flag will soon be removed. If you do not want any filtering, you need to "+
+							"remove any filters you might have set, instead of passing the flag.",
+						EnableIngestionFilteringFlag,
+					)
+				}
+				return nil
```

should you return an error here, so horizon does not start up, and the user will then be informed to the need to remove the flag usage?